node/network/statement-distribution/src/vstaging/tests/grid.rs

```diff
@@ -14,7 +14,252 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
-// TODO [now]: backed candidate leads to advertisement to relevant validators with relay-parent
+use super::*;
+
+use polkadot_node_network_protocol::grid_topology::TopologyPeerInfo;
+use polkadot_primitives_test_helpers::make_candidate;
+
+// Backed candidate leads to advertisement to relevant validators with relay-parent.
+#[test]
+fn backed_candidate_leads_to_advertisement() {
```

Comment:
@rphmeier any idea why no advertisements are being sent to the grid in this test? I must be missing something but I'm not sure what.

Reply:
StatementDistributionMessage::Backed needs to be received by the subsystem before that happens.